packages/babel-preset-typescript/src/plugin-rewrite-ts-imports.ts

```diff
@@ -31,22 +56,28 @@ export default declare(function ({ types: t, template }) {
           | t.ExportAllDeclaration
           | t.ExportNamedDeclaration
         >,
+        state,
       ) {
         const node = path.node;
         const kind = t.isImportDeclaration(node)
           ? node.importKind
           : node.exportKind;
         if (kind === "value") {
-          maybeReplace(node.source, path.get("source"));
+          maybeReplace(node.source, path.get("source"), state);
         }
       },
-      CallExpression(path) {
+      CallExpression(path, state) {
         if (t.isImport(path.node.callee)) {
```

Comment:
While we are changing this, could you wrap this in a `!process.env.BABEL_8_BREAKING`?